### PR TITLE
ENG-1120 Added ability to extract public file browser tree

### PR DESCRIPTION
**Pull request by ffleandro**

### CODE REVIEW

**Review by w-caffiero-entando — COMMENTED** [review]
@ffleandro in serveral points there is commented code related to the old key "cms_asset"

File: lib/env-bundler.js [hunk 1]
```diff
@@ -282,9 +340,9 @@ const collectQuestions = [
       { name: 'Microservices', value: 'plugin' },
       { name: 'Content Templates', value: 'contentModel' },
       { name: 'Content Types', value: 'contentType' },
-      { name: 'Resources', value: 'resource' },
       { name: 'Groups', value: 'group' },
       { name: 'Categories', value: 'category' },
+      { name: 'File Browser', value: 'resource' },
```
Comment (w-caffiero-entando) [inline lib/env-bundler.js:345]: @ffleandro Assets?

**Review by Kerruba — APPROVED** [review]
LGTM

File: lib/env-bundler.js (hunk 1 above)
Comment (ffleandro) [reply lib/env-bundler.js:345]: It's the name of the descriptor/folder inside the bundle. In the case of the File Tree, everything goes into a `resources` folder.

**Pull request merged**